benchmark/utils/DriverUtils.swift

```diff
@@ -227,27 +231,53 @@ func stopTrackingObjects(_: UnsafeMutableRawPointer) -> Int
 #endif
 
 class SampleRunner {
```

Comment:
Fixed again in https://github.com/apple/swift/pull/7810/commits/65f05b3e9cadd183a94a38422d98c00c8bb0dbbf